conf/local.conf.sample

```diff
@@ -74,7 +74,7 @@ CORE_IMAGE_EXTRA_INSTALL += "${@bb.utils.contains('IMAGE_FEATURES', 'read-only-r
 #ROOT_PASSWORD = "0"
 IMAGE_CLASSES:append = " image-root-password"
 
-# User features lets you manipulate the distro features. To add a distro
+# User features lets more easily manipulate the distro features. To add a distro
```

Comment:
User feature allows easier manipulation of the distro features

"lets more easily" sounds weird IMO.

Reply:
Yes, I have no idea how this grammar got mucked up, thanks.